packages/subgraph/tests/token/governance-erc20.test.ts

```diff
@@ -279,5 +278,61 @@ describe('Governance ERC20', () => {
       member.assertEntity();
       assert.entityCount('TokenVotingMember', 1);
     });
+
+    test('it should move delegation between members and end up deleting th member with no balances or delegated tokens', () => {
+      let memberOneAddress = ADDRESS_ONE;
```

Comment:
i would split this into 2 test
one: to test delegates moves between 2 memebers
two: to test if memeber is deleted once conditions are met

Reply:
+1

Reply:
okay, the delegation is tested in prior tests, I left this one to check if member has 0 balance and 0 voting power i gets deleted